<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1984</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>441</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Automatic Speech Recognition</string>
    </property>
    <widget class="QComboBox" name="microphoneMBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Microphone</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="startASR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearASR">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopASR">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="asrTextBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>351</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ASR</string>
     </property>
    </widget>
    <widget class="QComboBox" name="recognizerMBOX">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="googleGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>421</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Google Options</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
     <widget class="QComboBox" name="googleLanguage">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>API</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Endpoint</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="googleAPI">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="googleEndPoint">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="whisperGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Whisper Options</string>
     </property>
     <widget class="QComboBox" name="whisperModel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>72</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
     <widget class="QComboBox" name="whisperDevice">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>72</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="whisperLanguage">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>72</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1984</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
